# Sampling protocol for Dark Genes Meta Analysis

## Overview:
1. [**Confirm study is appropriate**](#confirm)
2. [**Locate files and metadata**]((#locate)
3. [**Input comparison metadata into “RNAseq_Data” spreadsheet**](#data)
4. [**Input sample metadata into “RNAseq_Samples” spreadsheet**](#samples)
5. [**Example**](#example)  

## <a name="confirm"></a> Step 1: Confirm study is appropriate

1. Pull data from [this spreadsheet](https://docs.google.com/spreadsheets/d/1ScX6AoRWQlxoszbX36I-pMBD5XGWKrxVqukirGzGOcI/edit#gid=0) on the "Studies" tab
2. Confirm it has not been completed (cells are not colored)
3. Search and download the paper from [Google Scholar](https://scholar.google.com/)
4. Once you have the paper open, make sure it completes **ALL** of the following criteria:
  * Includes corals or other cnidarians
  * Uses RNA-seq methods (not tag-seq or microsatellites)
  * Has a clear experimental “control” and “heated” sample comparison
5. If it **does not** meet the above criteria, highlight the whole row red and move on to the next study
6. If it **does** meet the above criteria, move on to Step 2!

## <a name="locate"></a> Step 2: Locate files and metadata


Can find samples

## <a name="data"></a> Step 3: Input comparison metadata into “RNAseq_Data” spreadsheet


### DOI


### Reference


### Comparison_ID


### Species
### Sample_Location
### Coordinates
### Life Stage
### High_temp
### Ambient_Temp
### Ramp_up_rate
### Acclimation_Period
### Duration_inDay
### Replicates
### Num_Genotype
### Method_RNA
### Other_Stressors
### NCBI_BioProject
### Data_File_Link
### Paper_Link
### Other_Comments

## <a name="samples"></a> Step 4: Input sample metadata into “RNAseq_Samples” spreadsheet


### Treatment_ID
### Comparison_ID
### Study_Number
### DOI
### Reference
### Species
### TaxID
### Sample_Location
### Coordinates
### Life_Stage
### Library_Name
### Accession_Experiment
### Temp
### pH
### Symbiont_Type
### Download_path

## <a name="example"></a> Example
